packages/react-instantsearch-hooks-dom/src/ui/RefinementList.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import { cx } from './lib/cx';
+
+import type { RefinementListItem } from 'instantsearch.js/es/connectors/refinement-list/connectRefinementList';
+
+export type RefinementListProps = React.HTMLAttributes<HTMLDivElement> & {
+  items: RefinementListItem[];
+  onRefine(item: RefinementListItem): void;
+  searchBox?: React.ReactNode;
```

Comment:
No it's just a node, just like `children`. That's the "slot-based" approach I was describing in the description; instead of having the search box in the UI component directly and pass a ton of props I'm composing by passing the full thing down from the widget. Same goes for `noResults` and `showMoreButton`.

Not sure this is the right approach yet but the alternative meant passing down a ton of props, this felt cleaner.